### PR TITLE
Update scheduler configuration

### PR DESCRIPTION
**Pull request by alicjak1519**

### CODE REVIEW

**Comment by alicjak1519** [comment]
I don't know exactly why, but it works. 
According to the docs of the newest pl version, we should use [trainer.estimated_stepping_batches](https://pytorch-lightning.readthedocs.io/en/latest/api/pytorch_lightning.trainer.trainer.Trainer.html#pytorch_lightning.trainer.trainer.Trainer.estimated_stepping_batches) property, but in our case, it doesn't work (I updated PL version locally to test it). 
Probably it is connected with additional sanity check steps (?) but in spite of using [trainer = Trainer(num_sanity_val_steps=0)](https://forums.pytorchlightning.ai/t/lr-scheduler-onecyclelr-valueerror-tried-to-step-x-2-times-the-specified-number-of-total-steps-is-x/259/3), OneCycleLR scheduler doesn't work neither. :( It work's properly for the fix I proposed in this PR, but I'm not able to explain it well. :(

**Review by AgaMiko — APPROVED** [review]
